src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
+  } else if (days >= 7) {
+    totalValue = days * 40 - 50;
+  }
+
+  return totalValue;
```

Comment:
Use 'const' instead of 'let' for 'totalValue' since its value is not being reassigned but recalculated based on conditions.